libbeat/kibana/fields_transformer.go

```diff
@@ -115,6 +121,7 @@ func (t *fieldsTransformer) transformFields(commonFields common.Fields, path str
 
 func (t *fieldsTransformer) update(target *common.MapStr, override common.Field) error {
 	field, _ := transformField(t.version, override)
+	return nil
```

Comment:
@graphaelli @simitt I get the following error if I don't skip the duplicated check:

```
Exiting: error generating index pattern: field <metricset.module> is duplicated, remove it or set 'overwrite: true', {Name:metricset.module Type:keyword Description:Name of the module this data is coming from.
This information is coming from the modules used in Beats or Logstash.
```

I'm trying to figure out why this is the case as metricbeat.module is as far as I can see not defined twice. Any idea?

Reply:
I tried to run `make update` on metricbeat, but I cannot reproduce this error.

Reply:
It does not happen on `make update` but only when running `./metricbeat export index-template`. That is the strange thing (or probably the key hint why it does not work)

Reply:
It looks like everything defined in the `metricbeat/_meta/fields.common.yml` is also defiined in the created `libbeat/fields.yml`, and both of them are used for the index pattern creation.